RELEASE_NOTES.rst

```diff
@@ -61,6 +63,15 @@ Exitcode changes:
 
 Bugfixes:
 
```

Comment:
you should include https://github.com/coala-analyzer/coala/commit/331406c21966cfc5284101156f2075bf40264340